utils_test.py

```diff
@@ -60,23 +68,25 @@ def test_to_ascii(self):
         self.assertEqual(parsed_str, b'')
 
     def test_yaml_dict_conversion(self):
+        # type: () -> None
         """Test yaml_from_dict and dict_from_yaml methods."""
         test_dicts = [{}, {'a': 'b'}, {'a': 2}, {'a': ['b', 2, {'c': 3.5}]}]
 
         for adict in test_dicts:
-            yaml_str = python_utils.yaml_from_dict(adict)
+            yaml_str = python_utils.yaml_from_dict(adict) # type: ignore[no-untyped-call]
             yaml_dict = utils.dict_from_yaml(yaml_str)
             self.assertEqual(adict, yaml_dict)
 
-        with self.assertRaisesRegexp(
+        with self.assertRaisesRegexp( # type: ignore[no-untyped-call]
             utils.InvalidInputException,
             'while parsing a flow node\n'
             'expected the node content, but found \'<stream end>\'\n'):
             yaml_str = utils.dict_from_yaml('{')
 
     def test_recursively_remove_key(self):
+        # type: () -> None
         """Test recursively_remove_key method."""
-        d = {'a': 'b'}
+        d = {'a': 'b'} # type: Any
```

Comment:
Should we try to not use `Any`?

Reply:
I think in test files, it will be best to use Any because a variable can be assigned to a lot of value types. What do you think?

Reply:
@vojtechjelinek I have suggested @hardikkat24 the same and we have also active discussion in [this](https://docs.google.com/document/d/1P6DkYljDaGp3I5DzkGMIFDICKaQ6QJI7wa6B-1wUKOs/edit?disco=AAAAMiOSqXk) thread of Type Annotations Draft Wiki doc.

Reply:
Changed the implementation and broke the test into multiple tests as per the discussion on the above thread.